pyconcz_2016/settings.py

```diff
@@ -83,18 +83,21 @@
 # Database
 # https://docs.djangoproject.com/en/1.9/ref/settings/#databases
 
-RDS_USER = os.environ['RDS_USER']
-
-DATABASES = {
-    'default': {
-        'ENGINE': 'django.db.backends.postgresql_psycopg2',
-        'HOST': os.environ['RDS_HOST'],
-        'NAME': os.environ.get('RDS_NAME', RDS_USER),
-        'USER': RDS_USER,
-        'PASSWORD': os.environ['RDS_PASSWORD'],
-        'PORT': os.environ.get('RDS_POST', 5432),
+try:
+    RDS_USER = os.environ['RDS_USER']
```

Comment:
RDS_USER and RDS_HOST are both required, so these errors shouldn't be silenced. Empty `DATABASES` config shouldn't be allowed in production settings.